README.md

```diff
@@ -33,7 +33,7 @@ For command line help, visit our wiki page on Bitters’ [command line interface
 
   *If you use [rbenv](https://github.com/sstephenson/rbenv), be sure to run `rbenv rehash` without any errors.*
 
-2. Install [Bourbon](https://github.com/thoughtbot/bourbon#installation) (required) and [Neat](https://github.com/thoughtbot/neat#installation) (optional).
+2. Install [Bourbon](https://github.com/thoughtbot/bourbon#installation) (required)
```

Comment:
No, we are also using Bourbon for `#{$all-text-inputs}`, `#{$all-buttons}` and `$helvetica`.